docs/advanced-topics/tools-vs-workflows.rst

```diff
@@ -22,43 +22,45 @@ Legacy Dockstore tools are more associated with creating/owning Docker images th
-+------------------------+------------------------------------------+-------------------------------------------------+-------------------------------------------------+
-
-What about workflows that run in Docker containers?
++----------------------+-----------------------------+----------------------------------------+-------------------------------------------------+
+| Dockstore Support    | Legacy Tool                 | GitHub App Tool                        | Workflow                                        |
++======================+=============================+========================================+=================================================+
+| Dockstore CLI        |  run in tool mode           |  run in workflow mode                  | run in workflow mode                            |
++----------------------+-----------------------------+----------------------------------------+-------------------------------------------------+
+| Launch with buttons  |  currently not supported    |  currently not supported               | supported                                       |
++----------------------+-----------------------------+----------------------------------------+-------------------------------------------------+
+| Versioning           |  based off of image's tags  |  based off of branches/tags from git   | based off of branches/tags from git repository, |
+|                      |                             |                                        |                                                 |
+|                      |                             |  repository, even if one of the tasks  | even if one of the tasks executes in a docker   |
+|                      |                             |                                        |                                                 |
+|                      |                             |  executes  in a docker image.          |                                                 |
++----------------------+-----------------------------+----------------------------------------+-------------------------------------------------+
+
+what about workflows that run in docker containers?
 ---------------------------------------------------
 
-Workflow descriptor files do not require a Docker container. That being said, you can still specify external Docker images
-within the descriptor files, and doing so is generally recommended. Each task (WDL) or step (CWL) can specify a Docker image, which might be totally different from the Docker image used by a previous task/step in the workflow. Because of this, workflows registered on Dockstore that reference Docker image(s) will still follow versioning from GitHub, instead of versioning based on the tags of the Docker image(s).
+workflow descriptor files do not require a docker container. that being said, you can still specify external docker images
```

Comment:
I think something went wrong with a macro or other wizardry used for quick edits here -- seems it's only here I'll just make a push that fixes them.